tests/test-sources/modules/performance/byte-compile-lua.nix

```diff
@@ -325,70 +327,62 @@ in
 
   # performance.byteCompileLua.luaLib for propagatedBuildInputs
   lua-lib-propagated-build-inputs =
-    {
-      config,
-      lib,
-      pkgs,
-      ...
-    }:
+    { config, lib, ... }:
```

Comment:
Why?

Reply:
Looks like the removed `pkgs` module arg was shadowing the `pkgs` test-file arg?

Reply:
> Looks like the removed pkgs module arg was shadowing the pkgs test-file arg?
No, I'm sure that I removed it after it became unused. Look at the commit:
https://github.com/nix-community/nixvim/pull/3314/commits/237d659c55e075c1274de9aa6b440c078f1a149c
After this there is no pkg usage in this test.

Reply:
Ah, thank's for clarifying. Hard to spot that kinda thing without an LSP!